src/lib.rs

```diff
@@ -0,0 +1,126 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+use futures::FutureExt;
+use tokio::{
+    io::{AsyncReadExt, AsyncWriteExt},
+    net::{TcpListener, TcpStream},
+    sync::broadcast,
+};
```

Comment:
Can you please keep the original non-nested formatting?

Reply:
> Can you please keep the original non-nested formatting?

Sure, this is the `group_imports` rustfmt option btw